tests/ci/cherry_pick.py

```diff
@@ -58,27 +58,33 @@ class ReleaseBranch:
     CHERRYPICK_DESCRIPTION = """Original pull-request #{pr_number}
 
 This pull-request is a first step of an automated backporting.
-It contains changes like after calling a local command `git cherry-pick`.
+It contains changes similar to calling a local command `git cherry-pick`.
 If you intend to continue backporting this changes, then resolve all conflicts if any.
 Otherwise, if you do not want to backport them, then just close this pull-request.
 
 The check results does not matter at this step - you can safely ignore them.
 
 ### Note
 
-This pull-request will be merged automatically as it reaches the mergeable state, \
-**do not merge it manually**. It's 100% safe, but completely meaningless.
+This pull-request will be merged automatically. Please, **do not merge it manually** \
+(but if you accidentally did, nothing bad will happen).
 
-### If the PR was closed and then reopened
+### If the PR was manually reopened after being closed
 
-If it stuck (e.g. for a day), check {pr_url} for `{backport_created_label}` *label* and \
-delete it if necessary. Manually merging will do nothing, since \
-`{backport_created_label}` *label* prevents the original PR {pr_url} from being \
-processed.
+If this PR is stuck (i.e. not automatically merged after one day), check {pr_url} for \
+`{backport_created_label}` *label* and delete it.
 
-If the cherry-pick PR is completely screwed, and you want to recreate it: delete the \
-`{label_cherrypick}` label and delete this branch.
-You may also need to delete the `{backport_created_label}` label from the original PR.
+Manually merging will do nothing. The `{backport_created_label}` *label* prevents the \
+original PR {pr_url} from being processed.
+
+If this cherry-pick PR is completely screwed by a wrong conflicts resolution, and you \
```

Comment:
Did you mean "If this cherry-pick PR was merged but the conflicts were wrongly resolved, you can recreate it."?

Reply:
nope, in this case, it's even more complex. PRs and branches for both cherry-pick and backport parts should be removed.